plugins/mcreator-core/datalists/mapcolors.yaml

```diff
@@ -1,62 +1,188 @@
-- DEFAULT
-- AIR
-- GRASS
-- SAND
-- CLOTH
-- TNT
-- ICE
-- IRON
-- FOLIAGE
-- SNOW
-- CLAY
-- DIRT
-- STONE
-- WATER
-- WOOD
-- QUARTZ
-- ADOBE
-- MAGENTA
-- LIGHT_BLUE
-- YELLOW
-- LIME
-- PINK
-- GRAY
-- SILVER
-- CYAN
-- PURPLE
-- BLUE
-- BROWN
-- GREEN
-- RED
-- BLACK
-- GOLD
-- DIAMOND
-- LAPIS
-- EMERALD
-- NETHERRACK
-- WHITE_TERRACOTTA
-- ORANGE_TERRACOTTA
-- MAGENTA_TERRACOTTA
-- LIGHT_BLUE_TERRACOTTA
-- YELLOW_TERRACOTTA
-- LIME_TERRACOTTA
-- PINK_TERRACOTTA
-- GRAY_TERRACOTTA
-- LIGHT_GRAY_TERRACOTTA
-- CYAN_TERRACOTTA
-- PURPLE_TERRACOTTA
-- BLUE_TERRACOTTA
-- BROWN_TERRACOTTA
-- GREEN_TERRACOTTA
-- RED_TERRACOTTA
-- BLACK_TERRACOTTA
-- CRIMSON_NYLIUM
-- CRIMSON_STEM
-- CRIMSON_HYPHAE
-- WARPED_NYLIUM
-- WARPED_STEM
-- WARPED_HYPHAE
-- WARPED_WART
-- DEEPSLATE_GRAY
-- RAW_IRON_PINK
-- LICHEN_GREEN
\ No newline at end of file
+- DEFAULT:
+  readable_name: "Default"
+- AIR:
+  readable_name: "None"
+  texture: IRON_BARS
+- GRASS:
+  readable_name: "Grass"
+  texture: GRASS
+- SAND:
+  readable_name: "Sand"
+  texture: SAND#0
+- CLOTH:
+  readable_name: "Wool"
+  texture: CANDLE#0

```

Comment:
Wool with candle texture does not sound ok to me. Please check if there are other mistakes like this one

Reply:
> Note: The icon is an example of a block using this color. So some colors might not match their name (e.g. WOOL using the white candle icon (white wool uses SNOW))

If you have another idea for the name, I'll take it because honestly I had no idea. w
When looking at which blocks use it, it makes no sense (e.g. white candle, white carpet, but not white wool, etc.)

Reply:
Hmm, is there any full-cube block using Wool? I would use that one in this case

Reply:
I would need to check again the code because I don't remember which blocks are using it.